### ISSUE TITLE
[Feedback] Page: /api/latest/continuous-aggregates/hypertable_detailed_size/

### ISSUE THREAD

**Issue by Solomon-Hill** [timescale/docs#2392]
### Is it easy to find the information you need?

None

### Are the instructions clear?

None

### How could we improve the Timescale documentation site?

incorrect info, the functions hypertable_size & hypertable_detailed_size return null when run against a CA. which is very annoying

**Comment by jonatas** [comment]
Hi @Solomon-Hill, I know what you mean.

@Loquacity I'll create a brief explanation of how to get it and let's move on to improve our docs based on this info.

The problem is that internally, we have different tables to store information about continuous aggregates and hypertables. 

For some reason we have both api docs for [continuous-aggregates/hypertable_detailed_size/](https://docs.timescale.com/api/latest/continuous-aggregates/hypertable_detailed_size/) and [hypertable/hypertable_detailed_size/](https://docs.timescale.com/api/latest/hypertable/hypertable_detailed_size/) but we don't have too functions.

In the continuous_aggregates page we should show people how to create a function for it:

```sql
CREATE OR REPLACE FUNCTION caggs_detailed_size(p_view_name TEXT, p_view_schema TEXT default 'public')
RETURNS TABLE(table_bytes bigint, index_bytes bigint, toast_bytes bigint, total_bytes bigint, node_name name)
AS $$
DECLARE
  v_hypertable_name TEXT;
  v_schema_name TEXT;
BEGIN
  SELECT cagg.hypertable_name, cagg.hypertable_schema
  INTO v_hypertable_name, v_schema_name
  FROM timescaledb_information.continuous_aggregates cagg
  WHERE cagg.view_name = p_view_name
  AND   cagg.view_schema = p_view_schema;

  RETURN QUERY
  SELECT * FROM hypertable_detailed_size(v_schema_name || '.' || v_hypertable_name);
END;
$$ LANGUAGE plpgsql;
```

Maybe we should also make the official function works with both types of arguments ?

**Comment by jonatas** [comment]
@noctarius solved the issue on 2.10 🎉 

 https://github.com/timescale/timescaledb/pull/5159
